ios/FluentUI/Calendar/CalendarConfiguration.swift

```diff
@@ -24,7 +24,7 @@ open class CalendarConfiguration: NSObject {
     /// By default, this is the default 'referenceStartDate' plus 10 years.  If overridden, make sure it's after (greater than) the 'referenceStartDate'
     @objc open var referenceEndDate: Date
 
-    @objc init(calendar: Calendar = .current) {
+    @objc public init(calendar: Calendar = .current) {
```

Comment:
I think I needed to add this in order to make a CalendarConfiguration object in the demoController

Reply:
now that this is public we need to add documentation on this class

Reply:
or comand + option + / in xcode shortcut to follow the latest syntax of documentation

Reply:
@harrieshin the documentation I added is in this commit: https://github.com/microsoft/fluentui-apple/pull/1007/commits/f13ea87307f045b225f5724866f8acccb9384f22. I added a comment above the class declaration. Were you also expecting documentation on the init method?

Reply:
Never mind actually - looked at other @objc public init methods and noticed they all have comments. Will add